README.md

```diff
@@ -1,4 +1,4 @@
-[![Netlify Status](https://api.netlify.com/api/v1/badges/e55b8400-46c3-44b0-96d1-cb101549dd1a/deploy-status)](https://app.netlify.com/sites/create-groups/deploys)
+[![Netlify Status](https://api.netlify.com/api/v1/badges/e55b8400-46c3-44b0-96d1-cb101549dd1a/deploy-status)](https://app.netlify.com/sites/create-groups/deploys) [website](https://create-groups.netlify.app/)
 # Create Groups
 
 This projects helps people create groups based on the items provided.
```

Comment:
Add the website link here

[website](https://create-groups.netlify.app/)

e.g 
This projects helps people create groups based on the items provided. 
[Click here](https://create-groups.netlify.app/) to see the website

Reply:
i have done